application/modules/home/controllers/Home.php

```diff
@@ -43,4 +43,19 @@ public function index() {
 		// load view
 		$this->load_view('top', $this->data);
 	}
-}
\ No newline at end of file
+
+	/**
+	 * Test side ci
+	 *
+	 * @return void
```

Comment:
__[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed

<!-- Comment by [SideCI](https://sideci.com) -->